content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,253 @@
+
+Strive to have your validator always embody the following properties. The more of these properties your validator embodies, the more reasons validator list publishers have to include it on their UNLs.
+
+- Available
+
+    An ideal validator is always running and submitting validation votes for every proposed ledger. Strive for 100% uptime.
+
+- In agreement
+
+    An ideal validator's votes match the outcome of the consensus process as often as possible. To do otherwise could indicate that your validator's software is outdated, buggy, or intentionally biased. Always run the [latest `rippled` release](https://github.com/ripple/rippled/tree/master) without modifications. [Watch `rippled` releases](https://github.com/ripple/rippled/releases) to be notified of new releases.
+
+- Issuing timely votes
+
+    An ideal validator's votes arrive quickly and not after a consensus round has already finished. Meeting recommended `rippled` server [system requirements](system-requirements.html), which include having a fast internet connection, helps with this.
+
+- Identified
+
+    An ideal validator has a clearly identified owner. Providing [domain verification](#provide-domain-verification) is a good start. Ideally, XRP Ledger network UNLs include validators operated by different owners in multiple legal jurisdictions and geographic areas. This reduces the chance that any localized events could interfere with the impartial operations of trusted validators.
+
+At present, Ripple (the company) cannot recommend any validators aside from those in the [default validator list](https://github.com/ripple/rippled/blob/develop/cfg/validators-example.txt). However, we are collecting data on other validators and building tools to report on their performance. For metrics on validators, see the XRP Charts [Validator Registry](https://xrpcharts.ripple.com/#/validators).
+
+
+
+## Set Up Proxies to Help Protect Your Validator
+
+To help protect a validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks, set up one or more stock `rippled` servers as proxies between your validator and inbound and outbound traffic.
+
+**Note:** While these servers are acting as proxies, they are not web proxies for HTTP(S) traffic.
+
+<!-- { TODO: Future: add a recommended network architecture diagram to represent the proxy, clustering, and firewall setup: https://ripplelabs.atlassian.net/browse/DOC-2046 }-->
+
+1. [Enable validation](#enable-validation-on-your-rippled-server) on your `rippled` server.
+
+2. Set up stock `rippled` servers. For more information, see [Install rippled](install-rippled.html).
+
+3. Configure your validator and stock `rippled` servers to run in a [cluster](cluster-rippled-servers.html). ***TODO: JHA still working on verifying that the validator should be in the cluster.***
```

Comment:
Hi @nbougalis! I've checked with a few folks on this - but we'd like to get your expert call on this, please. When setting up proxy stock servers to help protect a validator from DDoS attacks, we tell the user to cluster the `rippled` servers - the proxy stock servers and the validator. Can you confirm that the validator should be included in the cluster? Thank you for any guidance you can provide!

Reply:
Confirmed with Nik that it is ok to include the validator in the cluster with the proxy stock servers.